### ISSUE TITLE
Problem with Download for all documents without using last_days

### ISSUE THREAD

**Issue by jckruse** [pytr-org/pytr#109]
**Description of the bug**
While using the following prompt everything works fine:

pytr dl_docs --last_days 360 TR_Docs

But if I wanted to download all documents without using the last_days parameter the terminal output shows the following lines:

17:17:27 Subscribing #22797 timeline transactions
17:17:27 Received error message: '22797 E {"errors":[{"errorCode":"UNKNOWN_ERROR","errorField":null,"errorMessage":"Unknown error","meta":{"source":"PLATFORM"}}]}'
17:17:27 ('22797', {'type': 'timelineTransactions', 'after': '1856f415-09cb-41d4-9cde-5c0b22177b4c'}, {'errors': [{'errorCode': 'UNKNOWN_ERROR', 'errorField': None, 'errorMessage': 'Unknown error', 'meta': {'source': 'PLATFORM'}}]})
17:17:27 Received #22797 timeline transactions
17:17:27 Subscribing #22798 timeline transactions
17:17:27 Received error message: '22798 E {"errors":[{"errorCode":"UNKNOWN_ERROR","errorField":null,"errorMessage":"Unknown error","meta":{"source":"PLATFORM"}}]}'
17:17:27 ('22798', {'type': 'timelineTransactions', 'after': '1856f415-09cb-41d4-9cde-5c0b22177b4c'}, {'errors': [{'errorCode': 'UNKNOWN_ERROR', 'errorField': None, 'errorMessage': 'Unknown error', 'meta': {'source': 'PLATFORM'}}]})
17:17:27 Received #22798 timeline transactions
17:17:27 Subscribing #22799 timeline transactions
17:17:27 Received error message: '22799 E {"errors":[{"errorCode":"UNKNOWN_ERROR","errorField":null,"errorMessage":"Unknown error","meta":{"source":"PLATFORM"}}]}'
17:17:27 ('22799', {'type': 'timelineTransactions', 'after': '1856f415-09cb-41d4-9cde-5c0b22177b4c'}, {'errors': [{'errorCode': 'UNKNOWN_ERROR', 'errorField': None, 'errorMessage': 'Unknown error', 'meta': {'source': 'PLATFORM'}}]})
17:17:27 Received #22799 timeline transactions
17:17:27 Subscribing #22800 timeline transactions

After canceling with Strg+C and confirming with Y I get:

Task was destroyed but it is pending!
task: <Task pending name='Task-3' coro=<WebSocketCommonProtocol.keepalive_ping() running at C:\Users\jkr\pipx\venvs\pytr\Lib\site-packages\websockets\legacy\protocol.py:1254> wait_for=<Future pending cb=[Task.task_wakeup()]>>
Task was destroyed but it is pending!
task: <Task pending name='Task-2' coro=<WebSocketCommonProtocol.transfer_data() running at C:\Users\jkr\pipx\venvs\pytr\Lib\site-packages\websockets\legacy\protocol.py:963> wait_for=<Future pending cb=[Task.task_wakeup()]> cb=[Task.task_wakeup(), _wait.<locals>._on_completion() at C:\Program Files\Python312\Lib\asyncio\tasks.py:534]>
Task was destroyed but it is pending!
task: <Task pending name='Task-4' coro=<WebSocketCommonProtocol.close_connection() running at C:\Users\jkr\pipx\venvs\pytr\Lib\site-packages\websockets\legacy\protocol.py:1301> wait_for=<Task pending name='Task-2' coro=<WebSocketCommonProtocol.transfer_data() running at C:\Users\jkr\pipx\venvs\pytr\Lib\site-packages\websockets\legacy\protocol.py:963> wait_for=<Future pending cb=[Task.task_wakeup()]> cb=[Task.task_wakeup(), _wait.<locals>._on_completion() at C:\Program Files\Python312\Lib\asyncio\tasks.py:534]>>
Exception ignored in: <coroutine object WebSocketCommonProtocol.close_connection at 0x000001FDF0BB1630>
Traceback (most recent call last):
  File "C:\Users\jkr\pipx\venvs\pytr\Lib\site-packages\websockets\legacy\protocol.py", line 1337, in close_connection
  File "C:\Users\jkr\pipx\venvs\pytr\Lib\site-packages\websockets\legacy\protocol.py", line 1355, in close_transport
  File "C:\Users\jkr\pipx\venvs\pytr\Lib\site-packages\websockets\legacy\protocol.py", line 1379, in wait_for_connection_lost
  File "C:\Program Files\Python312\Lib\asyncio\timeouts.py", line 145, in timeout
RuntimeError: no running event loop

Do you have any ideas, how to handle this bug

**Comment by cellradar** [comment]
Hi there! 

I've got the same problem. 

pytr dl_docs --last_days XXXX TR_Docs works fine for 1194 days. I opened my TR account about 1670 days ago. 

These messages are just looping. Just the timeline activity log is counting upwards:

19:14:31 Received #26 timeline activity log
19:14:31 Subscribing #27 timeline activity log
19:14:31 Received error message: '32 E {"errors":[{"errorCode":"UNKNOWN_ERROR","errorField":null,"errorMessage":"Unknown error","meta":{"source":"PLATFORM"}}]}'
19:14:31 ('32', {'type': 'timelineActivityLog', 'after': '7071a4a9-a23b-4840-81cb-23b1b945faeb'}, {'errors': [{'errorCode': 'UNKNOWN_ERROR', 'errorField': None, 'errorMessage': 'Unknown error', 'meta': {'source': 'PLATFORM'}}]})
19:14:31 Received duplicate event c3e9f4e1-fe5b-45af-a97f-4ab76facb21a
19:14:31 Received duplicate event 1336c0b7-9ece-4874-ae36-ffba6fac9e90
19:14:31 Received duplicate event 04ea7940-0c54-4e0a-a5c0-3d69dc9e4b8b
19:14:31 Received duplicate event 83a4cc37-941e-41a4-8b3e-a174eb1f0aa2
19:14:31 Received duplicate event 0b045c76-16c4-4a8e-bf73-bb6b1745e5f7
19:14:31 Received duplicate event c0daf784-fef5-4a3e-a0fb-f4677fa1ef52
19:14:31 Received duplicate event 781a2c57-cd1c-4e7c-a3c5-cf14ac55c59d
19:14:31 Received duplicate event f8d86931-5d11-4302-939b-c3baf0d4be24
19:14:31 Received duplicate event 977268e1-7015-4ea9-b57c-5a44cdce11c1
19:14:31 Received duplicate event 61e14986-f1bf-44ad-9fb7-88ad6265ed36
19:14:31 Received duplicate event cecc84b8-34c7-4ae7-944a-aaf642e1997c
19:14:31 Received duplicate event 0f2619a9-20c5-442b-ba87-e286153676a8
19:14:31 Received duplicate event 17005524-a80a-414a-bdbf-8baa04928152
19:14:31 Received duplicate event 5786c7b9-1546-4983-b508-d975e6044670
19:14:31 Received duplicate event 77dce20e-6036-4f30-86df-a1ec90aa8aa5
19:14:31 Received duplicate event 293aa3b6-5dab-47b4-8c86-6d8565cc7e4a
19:14:31 Received duplicate event bfd5e6a9-f292-4bb8-8e58-1c62dfa5ab8e
19:14:31 Received duplicate event b0d5d310-2a37-4509-8f9d-0b72b7aa169a
19:14:31 Received duplicate event 572ca30e-01c3-4e8a-8a29-67c17742964b
19:14:31 Received duplicate event e0f4f2e4-e562-41e0-9b0c-8ddaa9217fa3
19:14:31 Received duplicate event deb6a714-f409-4c88-a314-e9d460511cab
19:14:31 Received duplicate event 4cd04e1a-a0d9-4e65-b602-101a8dc86763
19:14:31 Received duplicate event bfe1d317-b6b6-4b2e-8269-4c006d67c91a
19:14:31 Received duplicate event 924777d1-dea2-4f9e-b4b1-1dcc54c6d5de
19:14:31 Received duplicate event 63bef644-65fb-4e9c-a928-6dc577e8e47d
19:14:31 Received duplicate event 78f5cedc-006a-41bb-a38a-728291a21912
19:14:31 Received duplicate event 8180eb53-ae56-48bb-b2ed-3366a4293881
19:14:31 Received duplicate event c734171c-2174-4283-88a6-1b428ebea98c
19:14:31 Received duplicate event ed65484e-41fb-4f74-8d51-8857ae76d4b0
19:14:31 Received duplicate event 7071a4a9-a23b-4840-81cb-23b1b945faeb
19:14:31 Received #27 timeline activity log
19:14:31 Subscribing #28 timeline activity log
19:14:31 Received error message: '33 E {"errors":[{"errorCode":"UNKNOWN_ERROR","errorField":null,"errorMessage":"Unknown error","meta":{"source":"PLATFORM"}}]}'
19:14:31 ('33', {'type': 'timelineActivityLog', 'after': '7071a4a9-a23b-4840-81cb-23b1b945faeb'}, {'errors': [{'errorCode': 'UNKNOWN_ERROR', 'errorField': None, 'errorMessage': 'Unknown error', 'meta': {'source': 'PLATFORM'}}]})

**Comment by cellradar** [comment]
Hi there! 

I've got the same problem. 

pytr dl_docs --last_days XXXX TR_Docs works fine for 1194 days. I opened my TR account about 1670 days ago. 

These messages are just looping. Just the timeline activity log is counting upwards:

19:14:31 Received #26 timeline activity log
19:14:31 Subscribing #27 timeline activity log
19:14:31 Received error message: '32 E {"errors":[{"errorCode":"UNKNOWN_ERROR","errorField":null,"errorMessage":"Unknown error","meta":{"source":"PLATFORM"}}]}'
19:14:31 ('32', {'type': 'timelineActivityLog', 'after': '7071a4a9-a23b-4840-81cb-23b1b945faeb'}, {'errors': [{'errorCode': 'UNKNOWN_ERROR', 'errorField': None, 'errorMessage': 'Unknown error', 'meta': {'source': 'PLATFORM'}}]})
19:14:31 Received duplicate event c3e9f4e1-fe5b-45af-a97f-4ab76facb21a
19:14:31 Received duplicate event 1336c0b7-9ece-4874-ae36-ffba6fac9e90
19:14:31 Received duplicate event 04ea7940-0c54-4e0a-a5c0-3d69dc9e4b8b
19:14:31 Received duplicate event 83a4cc37-941e-41a4-8b3e-a174eb1f0aa2
19:14:31 Received duplicate event 0b045c76-16c4-4a8e-bf73-bb6b1745e5f7
19:14:31 Received duplicate event c0daf784-fef5-4a3e-a0fb-f4677fa1ef52
19:14:31 Received duplicate event 781a2c57-cd1c-4e7c-a3c5-cf14ac55c59d
19:14:31 Received duplicate event f8d86931-5d11-4302-939b-c3baf0d4be24
19:14:31 Received duplicate event 977268e1-7015-4ea9-b57c-5a44cdce11c1
19:14:31 Received duplicate event 61e14986-f1bf-44ad-9fb7-88ad6265ed36
19:14:31 Received duplicate event cecc84b8-34c7-4ae7-944a-aaf642e1997c
19:14:31 Received duplicate event 0f2619a9-20c5-442b-ba87-e286153676a8
19:14:31 Received duplicate event 17005524-a80a-414a-bdbf-8baa04928152
19:14:31 Received duplicate event 5786c7b9-1546-4983-b508-d975e6044670
19:14:31 Received duplicate event 77dce20e-6036-4f30-86df-a1ec90aa8aa5
19:14:31 Received duplicate event 293aa3b6-5dab-47b4-8c86-6d8565cc7e4a
19:14:31 Received duplicate event bfd5e6a9-f292-4bb8-8e58-1c62dfa5ab8e
19:14:31 Received duplicate event b0d5d310-2a37-4509-8f9d-0b72b7aa169a
19:14:31 Received duplicate event 572ca30e-01c3-4e8a-8a29-67c17742964b
19:14:31 Received duplicate event e0f4f2e4-e562-41e0-9b0c-8ddaa9217fa3
19:14:31 Received duplicate event deb6a714-f409-4c88-a314-e9d460511cab
19:14:31 Received duplicate event 4cd04e1a-a0d9-4e65-b602-101a8dc86763
19:14:31 Received duplicate event bfe1d317-b6b6-4b2e-8269-4c006d67c91a
19:14:31 Received duplicate event 924777d1-dea2-4f9e-b4b1-1dcc54c6d5de
19:14:31 Received duplicate event 63bef644-65fb-4e9c-a928-6dc577e8e47d
19:14:31 Received duplicate event 78f5cedc-006a-41bb-a38a-728291a21912
19:14:31 Received duplicate event 8180eb53-ae56-48bb-b2ed-3366a4293881
19:14:31 Received duplicate event c734171c-2174-4283-88a6-1b428ebea98c
19:14:31 Received duplicate event ed65484e-41fb-4f74-8d51-8857ae76d4b0
19:14:31 Received duplicate event 7071a4a9-a23b-4840-81cb-23b1b945faeb
19:14:31 Received #27 timeline activity log
19:14:31 Subscribing #28 timeline activity log
19:14:31 Received error message: '33 E {"errors":[{"errorCode":"UNKNOWN_ERROR","errorField":null,"errorMessage":"Unknown error","meta":{"source":"PLATFORM"}}]}'
19:14:31 ('33', {'type': 'timelineActivityLog', 'after': '7071a4a9-a23b-4840-81cb-23b1b945faeb'}, {'errors': [{'errorCode': 'UNKNOWN_ERROR', 'errorField': None, 'errorMessage': 'Unknown error', 'meta': {'source': 'PLATFORM'}}]})
